### ISSUE TITLE
Loadpixels / updatepixels combo causes drawing to stop

### ISSUE THREAD

**Issue by vixo777** [processing/p5.js#3191]
#### Nature of issue?

- [X] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [X] Core/Environment/Rendering
- [ ] Data
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Other (specify if possible)

#### Which platform were you using when you encountered this?

- [ ] Mobile/Tablet (touch devices)
- [X] Desktop/Laptop
- [ ] Others (specify if possible)

#### Details about the bug: 

- p5.js version: v0.7.2
- Web browser and version:  Chrome Version 68.0.3440.106 (Official Build) (64-bit) 
- Operating System: Windows 10
- Steps to reproduce this:

I have a sample here (https://editor.p5js.org/vixo777/sketches/H13tfrJOm) to test fading using loadPixels and updatePixels and progressively setting the pixels colors to the background color. The issue is that in the draw loop, only one line is displayed by the "randomLines" function, even when is called multiple lines.
Steps to reproduce (note that changing the pixels to background color is commented out on purpose. Feel free to uncomment to test):

Run the sketch as is, only one line is displayed
Comment out the call to "fadeImage()" and run. Multiple lines are displayed
Uncomment out the call to fadeImage and comment out the calls to either loadPixels or updatePixels, Multiple lines are displayed. Uncomment the calls to both load and update pixels. Only one line is displayed.
It would seem as if there is something in the combination of loadPixels and updatePixels that causes any additional drawing not to show up.

Hope this helps. Thanks.

**Comment by vixo777** [comment]
Looking at the code I'm understanding that updatePixels requires parameters to be provided (in my example I was calling them without parameters). I tried calling updatePixels(0,0,width,height) without results. Then I tried updatePixels(1,1,width, height) and it started working with the undesired effect of  scrolling diagonally, so finally I tried updatePixels(0,0,width-1, height-1) and it worked! (not sure it I'm missing the last pixel row and column). So, could it be there is a boundary bug in this call? is it supposed to be called with or without parameters? Thanks.
